src/components/VmActions/WindowsRdpButton.js

```diff
@@ -23,7 +23,7 @@ const WindowsRdpButton = ({ vm, className, config, onRDP }) => {
         href='#'
         key={vm.get('id')} id={`${idPrefix}-rdp`}
         className={className}
-        onClick={hrefWithoutHistory(onRDP(domain, username))}
+        onClick={(e) => { e.preventDefault(); onRDP(domain, username) }}
```

Comment:
Why do you use here `preventDefault`?

Reply:
Same exact thing as `hrefWithoutHistory` does, but without that function call.

Equivalent change with maintaining the helper function...
````javascript

  onClick={hrefWithoutHistory(e => onRDP(domain, username))}

````

I removed the helper function use because it tripped me up.

Reply:
`hrefWithoutHistory` is weird if we can just use `preventDefault`. +1.